pkg/types/baremetal/platform.go

```diff
@@ -33,10 +24,18 @@ type Platform struct {
 	// Default is qemu:///system
 	LibvirtURI string `json:"libvirtURI,omitempty"`
 
-	// IronicURI is the identifier for the Ironic connection.  It must be
-	// reachable from the host where the installer is run.
+	// ClusterProvisioningIP is the IP on the dedicated provisioning network
+	// where the baremetal-operator pod runs provisioning services,
+	// and http server to cache some downloaded content e.g RHCOS/IPA images
+	// These are used during the deployment of worker machines
 	// +optional
-	IronicURI string `json:"ironicURI,omitempty"`
+	ClusterProvisioningIP string `json:"provisioningHostIP,omitempty"`
+
+	// BootstrapProvisioningIP is the IP used on the bootstrap VM to
+	// bring up provisioning services that are used by terraform during the
```

Comment:
Can't have terraform in public API docs. `that are used to create the control-plane machines`

Reply:
ack will do